index.html

```diff
@@ -80,6 +80,13 @@ <h4>Setup</h4>
                 </td>
             </tr>
 
+            <tr>
+                <td><label for="wants_cdrom">Allow empty cd-rom drive</label></td>
+                <td>
+                    <input id="wants_cdrom" type="checkbox"><br>
+                </td>
+            </tr>
```

Comment:
Could you revert this for now? It's confusingly named, can cause problems with the state saving mechanism, and I'd like to eventually redo the UI anyway.

Reply:
Ok!